main.cpp

```diff
@@ -51,85 +51,95 @@ arm.flex();                             // 3)
 arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
-//  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  10)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  1) 
+//  Noun: cell phone
+//  action 1: receives call
+cellPhone.receivesCall();
+//  action 2: receives text
+cellPhone.receivesText();
+//  action 3: takes photos
+cellPhone.takesPhotos();
+
+//  2) 
+//  Noun: microphone
+//  action 1: receives audio
+microphone.receivesAudio();
+//  action 2: sends audio
+microphone.sendsAudio();
+//  action 3: filters audio
+microphone.filtersAudio();
+
+//  3)
+//  Noun: curtain
+//  action 1: blocks sunlight
+curtain.blocksSunlight();
+//  action 2: provides privacy
+curtain.providesPrivacy();
+//  action 3: supresses sound
+curtain.supressesSound();
+
+//  4)
+//  Noun: audio interface
+//  action 1:receives audio
+audioInterface.receivesAudio();
+//  action 2:sends audio
+audioInterface.sendsAudio();
+//  action 3:converts audio
+audioInterface.convertsAudio();
+
+//  5)
+//  Noun:pre amp
+//  action 1: receives audio
+preAmp.receivesAudio();
+//  action 2: sends audio
+preAmp.sendsAudio();
+//  action 3: converts audio
+preAmp.convertsAudio();
+
+//  6)
+//  Noun: rug
+//  action 1: protects floor
+rug.protectsFloor();
+//  action 2: covers floor
+rug.coversFloor();
+//  action 3: supresses sound
+rug.supressesSound();
+
+//  7) 
+//  Noun: coat
+//  action 1: provides warmth
+coat.providesWarmth();
+//  action 2: provides fashion
+coat.providesFashion();
+//  action 3: protects from weather
+coat.protectsFromWeather();
+
+//  8)
+//  Noun: drill
+//  action 1: inserts screws
+drill.insertsScrews();
+//  action 2: removes screws
+drill.removesScrews();
+//  action 3: makes holes
+drill.makesHoles();
```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.